frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditComponentIdRow/EditComponentIdRow.module.css

```
@@ -0,0 +1,13 @@
+.StudioTextfieldSchema {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.KeyVerticalIcon {
```

I believe that the class name shouldn't contain information about which icon it is; it's sufficient to indicate that it's an icon. Perhaps we could name the class 'prefixIcon'? This way, if the icon needs to change to something else later on, we wouldn't need to modify the class name.

```suggestion
.prefixIcon {
```